### PR TITLE
Bugfixes for parameterize_exp and simulation py

### PR DESCRIPTION
**Pull request by proteneer**
@chuantian 

This still uses a fixed random seed between epochs - we need to be able to remove it eventually

### CODE REVIEW

File: fe/parameterize_exp.py [hunk 1]
```diff
@@ -67,66 +69,49 @@ def get_masses(m):
...
-    all_du_dls = jnp.array(all_du_dls)
-
-    bkwd = all_du_dls[:, T:]
-    bkwd_sched = schedule[T:]
+            print("conf_idx", conf_idx, "dG None")
 
-    dG_bkwd = math_utils.trapz(bkwd, bkwd_sched) # integral from 0 to inf
-    dG_bkwd = -dG_bkwd
+    all_du_dls = jnp.array(all_du_dls)
+    work_insertion = math_utils.trapz(all_du_dls, schedule) # integral from 0 to inf
```
Comment (chuantian) [inline fe/parameterize_exp.py:87]: from inf to 0?

File: fe/parameterize_exp.py [hunk 2]
```diff
@@ -228,6 +168,13 @@ def error_fn(all_du_dls_raw, T, schedule, true_dG):
     host_conf = np.array(host_conf)
     host_name = "complex"
 
+    conformer = guest_mol.GetConformer(0)
```
Comment (chuantian) [inline fe/parameterize_exp.py:171]: this is the pre-supplied conformer

File: fe/parameterize_exp.py [hunk 3]
```diff
@@ -282,32 +270,27 @@ def error_fn(all_du_dls_raw, T, schedule, true_dG):
...
+        for trial_idx in range(args.num_trials):
 
-            v0 = np.zeros_like(x0)
+            if epoch % 10 == 0:
+                combined_pdb = Chem.CombineMols(Chem.MolFromPDBFile(host_pdb_file, removeHs=False), guest_mol)
+                combined_pdb_str = StringIO(Chem.MolToPDBBlock(combined_pdb))
+                out_file = os.path.join(args.out_dir, "epoch_"+str(epoch)+"_insertion_"+str(trial_idx)+".pdb")
+                writer = PDBWriter(combined_pdb_str, out_file)
+            else:
+                writer = None
```
Comment (chuantian) [inline fe/parameterize_exp.py:286]: we write PDB every 10 epochs to save storage

File: fe/parameterize_exp.py (hunk 1 above)
Comment (proteneer) [reply fe/parameterize_exp.py:87]: you're right - inf to zero.
